server/email/email-controller.js

```diff
@@ -0,0 +1,55 @@
+var mandrill = require('./email-service');
+var _ = require('underscore');
+
+module.exports = function(app, io){
+  var models = app.get('models');
+  var Gigs = models.Gig;
+  var UserGigs = models.UserGigs;
+  var Users = models.User;
+  var Locations = models.Location;
+
+  var emailController = {
+
+    sendEmails: function (req, res) {
+      var gigId = req.body.gigId;
+
+      Gigs.getGigInfo(gigId).then(function (gigInfo) {
```

Comment:
So in my first version I wasn't using the models.sequelize.Promise.map because we can just shoot the emails off one by one as we get the email addresses from the db. Do you want to refactor to this way for a particular reason? Is it faster/efficient or more readable? Thoughts?